Lib/ufo2ft/fontInfoData.py

```diff
@@ -55,9 +57,14 @@ def dateStringForNow():
 
 def openTypeHeadCreatedFallback(info):
     """
-    Fallback to now.
+    Fallback to the environment variable SOURCE_DATE_EPOCH if set, otherwise
+    now.
     """
-    return dateStringForNow()
+    if "SOURCE_DATE_EPOCH" in os.environ:
+        t = datetime.fromtimestamp(int(os.environ["SOURCE_DATE_EPOCH"]))
```

Comment:
you need to subtract the OpenType epoch (1st Jan 1904) from the SOURCE_DATE_EPOCH, because the latter uses the Unix epoch (1st Jan 1970).
Look at how @khaledhosny implemented it inside fontTools.misc.timeTools

Reply:
Oh sorry, my mistake. This is UFO openTypeHeadCreated, not the actual OpenType head.created field... :grin: 
It's correct like this

Reply:
Shouldn't dateStringForNow() include that, too? Just mimicking what that function returns.

Reply:
no, I just got confused, see comment above

Reply:
What would be nice also is a `try: except ValueError:` around the `int(os.environ["SOURCE_DATE_EPOCH"])` that catches when the value contained therein is invalid and logs a warning. I think it's better to warn and fall back to current time then a hard crash in this situation.

Reply:
actually nevermind. Even if we don't raise here, fonttools will crash later on if the SOURCE_DATE_EPOCH is invalid when computing the modified field. Keep it like that.